declarative-schedule-doc.md

```diff
@@ -339,6 +340,90 @@ scenarios:
 And on top of that you can use aliases to avoid duplication. For more details, check output
 [official documentation of openQA](http://open.qa/docs/)
 
+### 5. Use YAML_SCHEDULE_DEFAULT and YAML_SCHEDULE_FLOWS to YAML schedule
+
+
+We have 3 variables to help YAML scheduler to avoid conditional schedule and repetition. They are YAML_SCHEDULE_DEFAULT, YAML_SCHEDULE_FLOWS, YAML_SCHEDULE.
```

Comment:
we could say that "We have the possibility to combine 3 variables..."
Individually they don't solve the problem, in particular YAML_SCHEDULE doesn't.